aws_lambda_builders/workflows/go_modules/builder.py

```diff
@@ -50,6 +50,9 @@ def build(self, source_dir_path, output_path):
         if self.mode and self.mode.lower() == BuildMode.DEBUG:
             LOG.debug("Debug build requested: Setting configuration to Debug")
             cmd += ["-gcflags='all=-N -l'"]
+        else:
+            cmd += ["-ldflags='-s -w'"]
```

Comment:
@cgarvis 
Thanks for the PR! 
Can you also update the related unit test `tests/unit/workflows/go_modules/test_builder.py::TestGoBuilder::test_run_executes_bundler_on_nixes`?